src/main/webapp/WEB-INF/views/driver/add.jsp

```diff
@@ -0,0 +1,17 @@
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>Add driver to the car</title>
+</head>
+<body>
+<h1>Add driver to the car</h1>
+<h4 style="color: green">${message}</h4>
+<form method="post" action="${pageContext.request.contextPath}/cars/drivers/add">

```

Comment:
I think it is better to transfer this file to the car package and and create there one more package driver and  add this file.
car/driver/add
Check all naming create -add
display-all - all